src/app/admin/appointment/appointment-admin.component.ts

```diff
@@ -46,21 +44,15 @@ export class AppointmentAdminComponent {
    * @return {string}      Local hour in format 'HH:mm'
    */
   printHour(hour: number): string {
-    // add increment
-    const hourNew = this.calcHour(hour, this.addHours);
+    let hourNew = hour + this.increment * 100;
+    hourNew = hourNew < 0 || hourNew >= 2400 ? 0 : hourNew;
```

Comment:
No because it would then be displayed as being the same day at 02:00, meaning earlier than 23:00 (since the day is selected seperately)